editor/debugger/script_editor_debugger.cpp

```diff
@@ -1537,10 +1537,17 @@ void ScriptEditorDebugger::_error_activated() {
 		return;
 	}
 
-	TreeItem *ci = selected->get_first_child();
-	if (ci) {
-		selected->set_collapsed(!selected->is_collapsed());
+	// TreeItem *ci = selected->get_first_child();
+	// if (ci) {
+	// 	selected->set_collapsed(!selected->is_collapsed());
+	// }
+
+	Array meta = selected->get_metadata(0);
+	if (meta.size() == 0) {
+		return;
 	}
+
+	emit_signal(SNAME("error_activated"), String(meta[0]), int(meta[1]));
```

Comment:
This calls the signal with two arguments, but the signal is declared with a single argument

Reply:
Yes, makes sense. I adapted it from error_selected. I assume this one seems to have an unnecessary additional argument as well then?

Reply:
Looks like it, unsure